index.js

```diff
@@ -0,0 +1,60 @@
+const validatePassword = (userPassword) => {
+  if (userPassword.length < 8) return false
+  let passArray = userPassword.split('')
+  let hasNumb = false
+  let hasUpper = false
+  let hasLower = false
+  let hasSpec = false
+
+  for (let i = 0; i < passArray.length; i++) {
+    const currentChar = passArray[i]
```

Comment:
Remove this variable 

 if (specialCharCheck(passArray[i])) {
      hasSpec = true
      continue
    }